### PR TITLE
PR: "colour.SpectralDistribution_IESTM2714.read" method should probably return "self".

### PR DESCRIPTION
**Pull request by BPearlstine**
refs ticket #506 

changed the colour.SpectralDistribution_IESTM2714.read method to return self instead of True, updated instances of this call to match syntax in ticket.

### CODE REVIEW

**Comment by KelSolaar** [comment]
Hi @BPearlstine,

Did you start from the [develop](https://github.com/colour-science/colour/tree/develop) branch? It seems like this is stemming from master.

Cheers,

Thomas

**Comment by BPearlstine** [comment]
> Hi @BPearlstine,
> 
> Did you start from the [develop](https://github.com/colour-science/colour/tree/develop) branch? It seems like this is stemming from master.
> 
> Cheers,
> 
> Thomas

@KelSolaar 

I attempted to follow your docs on contributing as closely as I could, I may have made a mistake where it mentioned pulling in upstream master?

**Comment by KelSolaar** [comment]
Ah damn, my fault!

I will update the guide, this is a total oversight and it should be indeed `develop`, i.e. the repo default branch.

Do you think you could apply your changes on top of `develop` and force-push into your current branch? It should just be a matter of issuing `git rebase develop`, here is a guide: https://medium.com/@gabriellamedas/git-rebase-and-git-rebase-onto-a6a3f83f9cce

**Comment by BPearlstine** [comment]
> Ah damn, my fault!
> 
> I will update the guide, this is a total oversight and it should be indeed `develop`, i.e. the repo default branch.
> 
> Do you think you could apply your changes on top of `develop` and force-push into your current branch? It should just be a matter of issuing `git rebase develop`, here is a guide: https://medium.com/@gabriellamedas/git-rebase-and-git-rebase-onto-a6a3f83f9cce

@KelSolaar 
I'll get that pushed up this evening.

File: colour/io/tests/test_ies_tm2714.py [hunk 1]
```diff
@@ -239,9 +237,7 @@ def test_write(self):
         """
 
         sd_r = SpectralDistribution_IESTM2714(
-            os.path.join(RESOURCES_DIRECTORY, 'Fluorescent.spdx'))
-
-        sd_r.read()
+            os.path.join(RESOURCES_DIRECTORY, 'Fluorescent.spdx')).read()
 
         sd_r.path = os.path.join(self._temporary_directory, 'Fluorescent.spdx')
         self.assertTrue(sd_r.write())
```
Comment (KelSolaar) [inline colour/io/tests/test_ies_tm2714.py]: @BPearlstine : I think you need to modify the next line to `sd_t = SpectralDistribution_IESTM2714(sd_r.path).read()` and you are good! 👍

**Comment by KelSolaar** [comment]
Looks like it worked @BPearlstine ! 👍

**Pull request merged**